ui/site/css/streamer/_header.scss

```diff
@@ -15,7 +15,6 @@
     display: flex;
     flex-flow: column;
     justify-content: space-between;
-    width: 100%;
```

Comment:
This doesn't do anything atm as far as I can tell and messes up the layout when switching to "flex-direction: column" on small screens.